armips/move/battle_eff_seq/083.s

```diff
@@ -7,6 +7,8 @@
 .include "armips/include/monnums.s"
 .include "armips/include/movenums.s"
 
+PARENTAL_BOND_SUBSCRIPT equ (353)
```

Comment:
i may create a subseq definition file like `include/constants/battle_script_constants.h` for armips.  need to throw this in there instead

need to use lhea's script commands too